static/src/stylesheets/module/commercial/_facia-container.scss

```diff
@@ -76,13 +76,27 @@
     }
 }
 .commercial__title {
-    .i-guardian-logo-commercial {
+    @include fs-header(3);
+    line-height: 1;
+    min-height: $gs-baseline*3;
+    color: #ffffff;
+    fill: #ffffff;
+    margin: 0;
+
+    a {
+        color: #ffffff;
+    }
+
+    .i-guardian-logo-commercial,
+    .inline-guardian-logo-commercial, {
         display: inline-block;
 
         @include mq(leftCol) {
             display: none;
         }
     }
+    
+    .inline-marque-54,
     .i-marque-54 {
```

Comment:
i-marque-54 is still being used in the old fashioned way as it's in a js template as well, I'll try and get that sorted next.